pkg/handler/std_handler.go

```diff
@@ -21,8 +21,11 @@ func IndexPageHandler(svr server.Server) http.HandlerFunc {
 		location := r.URL.Query().Get("l")
 		tag := r.URL.Query().Get("t")
 		page := r.URL.Query().Get("p")
-		// @TODO: redirect to canonical
-		svr.RenderPageForLocationAndTag(w, location, tag, page, "landing.html")
+		dst := fmt.Sprintf("/Golang-%s-Jobs-In-%s", tag, location)
```

Comment:
it's a bit more complex than that. If tag or location are empty then the format is going to be different.

For eg.

scenario 1
tag = "DevOps"
location = "NYC"
dst = /Golang-DevOps-Jobs-In-NYC

scenario 2
tag = ""
location = "NYC"
dst = /Golang-Jobs-In-NYC

scenario 3
tag = "DevOps"
location = ""
dst = /Golang-DevOps-Jobs

scenario 4
tag = ""
location = ""
dst = /

Reply:
completely missed those variants 🤦‍♂